app/models/concerns/audit_extensions.rb

```diff
@@ -43,6 +44,17 @@ def self.humanize_class_name
 
   private
 
+  def filter_encrypted
+    self.audited_changes.each do |name,change|
+      next if change.nil? || change.to_s.empty?
+      if change.is_a? Array
+        change.map! {|c| c.to_s.start_with?(EncryptValue::ENCRYPTION_PREFIX) ? N_("[encrypted]") : c}
+      else
+        audited_changes[name] = N_("[encrypted]") if change.to_s.start_with?(EncryptValue::ENCRYPTION_PREFIX)
```

Comment:
Where does `[encrypted]` get translated in the UI? My last comment here suggested doing it in a UI helper, but I can't see that in this PR. (See https://github.com/theforeman/foreman/pull/3230#discussion_r81298367)